packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -244,6 +246,8 @@ jobs:
     with:
       package-name: ${package.name}
       working-directory: $repoRelativePath
+      ${hasGoldens ? 'runner-os: macos-latest' : ''}
```

Comment:
Remove, it's not a parameter you can pass anyway. But only `has-goldens` is needed to know which runner to use.